lib/beautiful/theme_assets.lua

```diff
@@ -228,13 +230,15 @@ function theme_assets.recolor_titlebar(theme, color, state, postfix)
         'titlebar_maximized_button_'..state..'_inactive',
         'titlebar_maximized_button_'..state..'_active',
```

Comment:
this whole block looks weird now after you added active/inactive as parameters

Reply:
Yes indeed, the problem was that close/minimize doesn't have the active/inactive variants (and I didn't want to change too much). With the current patch, we just filter the non-matching entries. Alternatively that table could be generated, instead of being "hardcoded" (and filtered). What do you think?

Reply:
>  Alternatively that table could be generated, instead of being "hardcoded" (and filtered). What do you think?

yup, that's what i was implying about

however, as you yourself mentioned:

>  the problem was that close/minimize doesn't have the active/inactive variants

so if you or @Elv13 or somebody else have some nicer solution in mind, i don't want to nitpicking too much and the current code would be fine

Reply:
Added a fixup commit with the table being generated. Not sure if this is overkill...